analog-rings/usr/share/asteroid-launcher/watchfaces/analog-rings.qml

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2018 - Timo Könnecke <el-t-mo@arcor.de>
+ *               2016 - Sylvia van Os <iamsylvie@openmailbox.org>
+ *               2015 - Florent Revest <revestflo@gmail.com>
+ *               2012 - Vasiliy Sorokin <sorokin.vasiliy@gmail.com>
+ *                      Aleksey Mikhailichenko <a.v.mich@gmail.com>
+ *                      Arto Jalkanen <ajalkane@gmail.com>
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// Asteroid theme colors
+// yellow	Qt.rgba(0.945, 0.769, 0.059, 1)
+// Orange	Qt.rgba(1, 0.549, 0.149, 1)
+// red		Qt.rgba(0.871, 0.165, 0.102, 1)
+
+import QtQuick 2.1
+
+Item {
+    property var radian: 0.01745329252
+
+    Image {
+        id: logoAsteroid
+        source: "asteroid_logo.png"
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.verticalCenter: parent.verticalCenter
+	width: parent.width/5.2
```

Comment:
As a general note, this kind of parent.width*0.x dimensions should use Dims.w(percentage) instead. parents.height*0.x should use Dims.h(percentage) and when a font size is being specified, Dims.l(percentage) should be used. For example parent.width*0.12 should be replaced by Dims.w(12). parent.width/5.2 should be Dims.w(19.2) etc...
Those functions are available when importing asteroid.controls 1.0.

Your watchfaces being great examples for future contributors, it's important to have best practices in mind.